### DISCUSSION TITLE
Orientation does not appear in fundamental region

### DISCUSSION THREAD

**Discussion by Debo-rah** [mtex-toolbox/mtex#1150] [Ask Anything] [answered]
Hi all,

some of my orientations are not plotted in the fundamental region, when looking from y direction, when they are right on the border. If its slightly off, it doesnt appear anymore (see image). However, isnt the definition of a fundamental region, that ALL equivalent orientations appear in it?

And another question- i wanted to plot the same orientations in a Polefigure,  also "looking from different directions" but didnt find how to do it. I mean, that once 001 axis is aligned with the x axis , once with y axis and once with z axis... It should be easy, but I dont find the command.

![image](https://user-images.githubusercontent.com/72927930/126618869-9ead98b6-662b-4dc5-91da-ebd04e562d72.png)

**Comment by kilir** [comment, marked as answer]
Hi,
which version of mtex are you using?
Cheers,
Rüdiger

**Reply by Debo-rah** [reply]
mtex-5.5.0 in matlab 2018b

I just realised I didnt update my skript to the newer mtex version, although i do have mtex 7  installed as well. i will try it with that version again.

**Reply by Debo-rah** [reply]
yep, in mtex 5.7 it works fine. Sorry.... And thanks for asking!

any hints for my question to plot the PDF from another viewing point?
